ground/gcs/src/plugins/plugins.pro

```diff
@@ -342,3 +342,9 @@ plugin_boards_aeroquad.subdir = boards_aeroquad
 plugin_boards_aeroquad.depends = plugin_coreplugin
 plugin_boards_aeroquad.depends = plugin_uavobjects
 SUBDIRS += plugin_boards_aeroquad
+
+# Brotronics
+plugin_boards_brotronics.subdir = boards_brotronics

```

Comment:
I wonder if it would be possible to have qmakelook in the board plugin directory and automatically compile everything it sees there. I'm not sure if that's a better strategy or not, because on the one hand it makes for more "magic", whereas right now it's very obvious what needs to be done to support a new board. On the other hand, it would be nice to minimize the number of codebase changes required to support a new target and this change seems purely formulaic.